### PR TITLE
Laser tag guns double as Taser tag guns

### PR DESCRIPTION
**Pull request by 9600bauds**
Using a laser tag gun in your hand sets the projectile to a laser tag electrode. For when you want to practice your yakety-fu with someone and lasers just aren't the same thing.
The taser tag electrodes are visually distinct from the regular taser electrode, they stand out a little less but I think that's a good thing honestly.
mspaint_2017-01-03_19-53-27

Scoring against someone of the opposing team of laser tag will now play a serotonin-releasing sound and add 1 to your gun's score! You can see your gun's score by examining it and clear it via a verb.

Surprisingly, turrets actually work with the taser modes.

:cl: 
 * rscadd: Laser tag guns can now be used in your hand to switch them into Taser tag guns (to practice your yaketying!). They now also have a hitsound and a score counter when hitting people wearing laser tag vests of the opposite color.

### CODE REVIEW

File: code/modules/projectiles/guns/energy/tag.dm [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	fire_sound = 'sound/weapons/Laser.ogg'
+	var/laser_projectile = /obj/item/projectile/beam/lasertag/blue
+	var/taser_projectile = /obj/item/projectile/beam/lasertag/omni
+	var/needed_vest = /obj/item/clothing/suit/bluetag
+
+/obj/item/weapon/gun/energy/tag/isHandgun()
+	return TRUE
+
+/obj/item/weapon/gun/energy/tag/proc/score()
+    playsound(get_turf(src), 'sound/weapons/quake.ogg', 60)
```
Comment (9600bauds) [inline code/modules/projectiles/guns/energy/tag.dm:25]: pleasure

**Comment by Intigracy** [comment]
Could you make hitting with a laser tag laser stop doing a stun and instead deal halloss damage?

**Comment by 9600bauds** [comment]
I guess that'd make sense but I'm still not sure
let's hear a couple other opinions

**Comment by Intigracy** [comment]
They're functionally the same regardless of mode atm famalam.

**Comment by Exxion** [comment]
Lasers have no travel time. Tasers do.
That's the point of the change.

**Comment by TheFriggenSawbones** [comment]
ok listen
what if
what if

the laser tag armor acts as a suit accessory or something
or what if you can place it ontop of plasmaman suits

i want to play lasertag really badly

**Comment by Shadowmech88** [comment]
Just chug burn meds.

**Comment by Ephx1** [comment]
Look, the plasma thing thinks he's people!

**Comment by Intigracy** [comment]
Oh right, I forgot how lasers worked there for a moment.

**Pull request merged**